docs/tables/aws_cost_by_account_daily.md

```diff
@@ -95,12 +98,62 @@ with ranked_costs as (
     period_start,
     unblended_cost_amount::numeric::money,
     rank() over(partition by linked_account_id order by unblended_cost_amount desc)
-  from 
+  from
     aws_cost_by_account_daily
 )
 select * from ranked_costs where rank <= 10;
 ```
 
 ```sql+sqlite
-Error: SQLite does not support the rank window function.
+with ranked_costs as (
+  select
+    linked_account_id,
+    period_start,
+    cast(unblended_cost_amount as real) as unblended_cost_amount,
+    rank() over (partition by linked_account_id order by unblended_cost_amount desc) as rnk
+  from
+    aws_cost_by_account_daily
+)
+select * from ranked_costs where rnk <= 10;
```

Comment:
@ParthaI could you please remove the sqlite query over here since it does not support rank window function?